packages/core/src/Stores/client-store.js

```diff
@@ -2571,7 +2571,9 @@ export default class ClientStore extends BaseStore {
     }
 
     get is_pending_proof_of_ownership() {
-        return this.account_status?.authentication?.needs_verification?.includes('ownership');
+        if (!this.account_status?.authentication) return false;
+        const { ownership, needs_verification } = this.account_status.authentication;
+        return needs_verification?.includes('ownership') || ownership?.status === 'verified';
```

Comment:
why do we set is_pending_proof_of_ownership to true when ownership status is verified?

Reply:
`is_pending_proof_of_ownership` flag is used for popups and for enabling the POO section. I modified the condition as per the requirement but kept the same flag variable name. May be I'll rename it